backend/supervisor/src/supervisor.ts

```
@@ -7,8 +7,9 @@
  * This file is part of the OperatorFabric project.
  */
 
-import express, {NextFunction} from 'express';
+import express, {NextFunction, Request, Response} from 'express';
 import {expressjwt, GetVerificationKey} from 'express-jwt';
+import {Request, Response} from 'express';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Request'. Consider to delete it or rename it.

'Request' is defined here:


(lint/suspicious/noRedeclare)